### ISSUE TITLE
No Response after installing on Manjaro Linux(Arch Based)

### ISSUE THREAD

**Issue by Binzzzz** [jasperproject/jasper-client#276]
sudo systemctl start jasper-voice-connrol
no response after executing this command in Arch Linux.

**Comment by Holzhaus** [comment]
Run
```
sudo systemctl status -l jasper-voice-control
```

**Comment by Binzzzz** [comment]
Jan 05 16:39:29 BinoyPC systemd[1]: Unit jasper-voice-control.service entered failed stae te.
Jan 05 16:39:29 BinoyPC systemd[1]: jasper-voice-control.service failed.

Output for above command . Any alternatives?

**Comment by Holzhaus** [comment]
I guess this is because you're running pulseaudio on your desktop PC. Unfortunately, an audio device can only be access by one process at a time. When logging in into your desktop environment, Pulseaudio starts up and gets a lock on your audio device.

The systemd service for jasper does not start jasper as your current user but as another user (`jasper`) and can't access your pulseaudio instance started during login. Thus a new pulseaudio instance for user `jasper` is started and tries to get a lock on the audio device, but the access is already blocked by the other instance.

TLDR; You can't start jasper with systemd/pulseaudio on desktop computer.

Instead, you can open a terminal and copy the config file to your current user:
```bash
$ mkdir ~/.jasper
$ cp /var/lib/jasper/.jasper/profile.yml ~/.jasper/
```

Now start Jasper as your current user:

```bash
$ jasper-voice-control
```

**Comment by Holzhaus** [comment]
Does is work now?

**Comment by Holzhaus** [comment]
No reply, closed.

**Comment by Binzzzz** [comment]
jasper-voice-control.service - Jasper Voice Control
   Loaded: loaded (/usr/lib/systemd/system/jasper-voice-control.service; disabled; vendor preset: disabled)
   Active: failed (Result: exit-code) since Tue 2015-01-13 09:27:07 IST; 18min ago
  Process: 1552 ExecStart=/usr/bin/jasper-voice-control (code=exited, status=1/FAILURE)
 Main PID: 1552 (code=exited, status=1/FAILURE)

Jan 13 09:27:07 BinoyPC jasper-voice-control[1552]: File "/usr/share/jasper-voice-control/client/conversation.py", line 3, in <module>
Jan 13 09:27:07 BinoyPC jasper-voice-control[1552]: from notifier import Notifier
Jan 13 09:27:07 BinoyPC jasper-voice-control[1552]: File "/usr/share/jasper-voice-control/client/notifier.py", line 4, in <module>
Jan 13 09:27:07 BinoyPC jasper-voice-control[1552]: from modules import Gmail
Jan 13 09:27:07 BinoyPC jasper-voice-control[1552]: File "/usr/share/jasper-voice-control/client/modules/Gmail.py", line 5, in <module>
Jan 13 09:27:07 BinoyPC jasper-voice-control[1552]: from dateutil import parser
Jan 13 09:27:07 BinoyPC jasper-voice-control[1552]: ImportError: No module named dateutil
Jan 13 09:27:07 BinoyPC systemd[1]: jasper-voice-control.service: main process exited, code=exited, status=1/FAILURE
Jan 13 09:27:07 BinoyPC systemd[1]: Unit jasper-voice-control.service entered failed state.
Jan 13 09:27:07 BinoyPC systemd[1]: jasper-voice-control.service failed.

**Comment by Binzzzz** [comment]
Sorry for the delay. I was out for few days. am getting the above result while executing

**Comment by Holzhaus** [comment]
`yaourt -S jasper-plugins`

**Comment by Binzzzz** [comment]
Jan 13 20:22:57 BinoyPC pulseaudio[5174]: [pulseaudio] module-jackdbus-detect.c: Unable to contact D-Bus session bus: org.freedesktop.DBus.Error.NotSupported: Unable to autolaunch a dbus-daemon without a $DISPLAY for X11
Jan 13 20:22:57 BinoyPC pulseaudio[5174]: [pulseaudio] module.c: Failed to load module "module-jackdbus-detect" (argument: "channels=2"): initialization failed.

**Comment by Holzhaus** [comment]
Is this your Desktop PC? Looks like you aren't running X11/dbus or tried this via SSH without exporting $DISPLAY first.

Please open the terminal inside your Desktop Environment and run jasper from there. If the problem persists, ask the guys in the [mageia forum](https://forums.mageia.org/en/viewforum.php?f=26) how to set up pulseaudio correctly.

**Comment by Binzzzz** [comment]
yes am running this in my PC. and am getting same problem again
